package.json

```diff
@@ -9,11 +9,12 @@
     "test": "mocha --require ts-node/register --require dotenv/config",
     "test-coverage": "nyc --reporter=text mocha --require ts-node/register src/__test__",
     "typecheck": "tsc --noEmit",
-    "build": "copyfiles -u 1 src/views/**/* src/views/**/**/* src/views/**/**/**/* dist/ && tsc",
+    "build": "copyfiles -u 1 src/views/**/* dist/ && tsc",
```

Comment:
`**` is a glob for all nested directories, so shouldn't be any need for the extra ones